### ISSUE TITLE
Slider: High-contrast slider without labels is extremely difficult to use

### ISSUE THREAD

**Issue by mpriour** [Esri/calcite-components#4514]
### Summary

As identified in the High contrast accessiblity audit (https://github.com/Esri/calcite-components/wiki/High-Contrast-Audit#slider-line), the unlabeled slider is very difficult to use.
The track is invisible and the handle is a very large box with low positional precision.

### Actual Behavior

No track is present and the handle(s) are large boxes.

<img width="405" alt="Screen Shot 2022-01-14 at 11 42 33 AM" src="https://user-images.githubusercontent.com/142636/167505294-5fa9f43d-319f-4974-94f1-4c9c07765ac6.png">

### Expected Behavior

Track should be visible and handles should indicate a more precise location.

<img width="410" alt="Screen Shot 2022-01-14 at 11 41 49 AM" src="https://user-images.githubusercontent.com/142636/167505477-f8db7a39-8e30-4b22-85b2-eb4204f3c9e3.png">

### Reproduction Sample

https://developers.arcgis.com/calcite-design-system/components/slider/

### Reproduction Steps

Windows
1. Set Windows color theme to a high-contrast variant.
2. Open the reproduction sample (https://developers.arcgis.com/calcite-design-system/components/slider/) in any browser.
3. Observe the issue.

Mac
1. Open the reproduction sample (https://developers.arcgis.com/calcite-design-system/components/slider/) in Chrome or Edge.
2. Open Dev Tools.
3. Click on Elements tab.
4. Next to the console sub tab, there should be a "Rendering" tab (if not you will need to enable it)
5. On the Rendering tab choose Forced-colors: active from the "Emulate forced colors..." selector
6. Observe the issue.

### Reproduction Version

@esri/calcite-components@1.0.0-latest

### Relevant Info

The track, handles, and handle extension all need forced-colors (high contrast) specific styling.

**Comment by geospatialem** [comment]
Verified in yesterday's release `1.0.0-beta.82`. **Great fix, @mpriour!** 🙌🏻 

Screenshot from Windows 10:
![high contrast slider fix on Windows](https://user-images.githubusercontent.com/5023024/167869397-69a39133-3b5f-4f0d-8db3-8b0c5593acaa.png)
